.github/workflows/fips-checksums.yml

```diff
@@ -73,3 +73,45 @@ jobs:
         with:
           name: fips_checksum
           path: artifact/
+
+  compute-abidiff:
+    runs-on: ubuntu-latest
+    steps:
+      - name: create build dirs
+        run: |
+          mkdir ./build-pristine
+          mkdir ./source-pristine
+          mkdir ./build
+          mkdir ./source
+          mkdir ./artifact
+      - name: install extra config support
+        run: sudo apt-get -y install libsctp-dev abigail-tools libzstd-dev zstd
+      - uses: actions/checkout@v4
+        with:
+          repository: ${{ github.event.pull_request.base.repo.full_name }}
+          ref: ${{ github.event.pull_request.base.ref }}
+          path: source-pristine
+      - name: config pristine
+        run: ../source-pristine/config --banner=Configured -g --strict-warnings enable-ktls enable-fips enable-egd enable-ec_nistp_64_gcc_128 enable-md2 enable-rc5 enable-sctp enable-ssl3 enable-ssl3-method enable-trace enable-zlib enable-zstd && perl configdata.pm --dump
+        working-directory: ./build-pristine
+      - name: make pristine
+        run: make -s -j4
+        working-directory: ./build-pristine
+      - uses: actions/checkout@v4
+        with:
+          path: source
+      - name: config
+        run: ../source/config --banner=Configured -g --strict-warnings enable-ktls enable-fips enable-egd enable-ec_nistp_64_gcc_128 enable-md2 enable-rc5 enable-sctp enable-ssl3 enable-ssl3-method enable-trace enable-zlib enable-zstd && perl configdata.pm --dump
```

Comment:
It may be worth setting this whole string to an environment variable to be used both in this case and the config pristine step so that updates to the config only need to be made in one place

Reply:
Done